src/main/java/wooteco/subway/auth/config/AuthenticationPrincipalArgumentResolver.java

```diff
@@ -5,36 +5,30 @@
 import org.springframework.core.MethodParameter;
 import org.springframework.web.bind.support.WebDataBinderFactory;
 import org.springframework.web.context.request.NativeWebRequest;
-import org.springframework.web.context.request.WebRequest;
 import org.springframework.web.method.support.HandlerMethodArgumentResolver;
 import org.springframework.web.method.support.ModelAndViewContainer;
-import wooteco.subway.auth.application.AuthService;
 import wooteco.subway.auth.domain.AuthenticationPrincipal;
 import wooteco.subway.auth.infrastructure.AuthorizationExtractor;
-import wooteco.subway.auth.infrastructure.JwtTokenProvider;
 import wooteco.subway.member.application.MemberService;
-import wooteco.subway.member.domain.Member;
 import wooteco.subway.member.dto.MemberResponse;
 
 public class AuthenticationPrincipalArgumentResolver implements HandlerMethodArgumentResolver {
-    private AuthService authService;
     private MemberService memberService;
 
-    public AuthenticationPrincipalArgumentResolver(AuthService authService, MemberService memberService) {
-        this.authService = authService;
+    public AuthenticationPrincipalArgumentResolver(MemberService memberService) {
         this.memberService = memberService;
     }
 
     @Override
     public boolean supportsParameter(MethodParameter parameter) {
         return parameter.hasParameterAnnotation(AuthenticationPrincipal.class);
     }
+
     @Override
     public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) {
```

Comment:
```suggestion
    public LoginMember resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) {
```
컨트롤러에서 해당 어노테이션을 사용해서 객체를 받는 경우, MemberResponse를 반환하는데, 오히려 request에 가까워 보이네요ㅎㅎ 다른 객체를 사용해보는건 어떨까요~?